src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_factory.py

```diff
@@ -99,7 +102,7 @@ def queue_data_service_factory(kwargs):
 
 
 def cloud_storage_account_service_factory(kwargs):
-    from azure.storage import CloudStorageAccount
+    CloudStorageAccount = get_sdk_attr('azure.multiapi.storage#CloudStorageAccount')
```

Comment:
How is "get_sdk_attr" different from "get_versioned_models"??  It seems like you can do the same thing with either.